charts/lily/values.yaml

```diff
@@ -31,7 +31,7 @@ daemon:
   # next time the pod is restarted. This may be useful if the daemon has fallen
   # out of sync for a long period of time.
   importSnapshot:
-    enabled: true
+    enabled: false
     url: https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car
```

Comment:
could a tool like like [`s3 sync`](https://docs.aws.amazon.com/cli/latest/reference/s3/sync.html) be used to download this snapshot more quickly?

Reply:
👍 I will release/merge this and add in a follow-on PR.

Reply:
https://github.com/filecoin-project/helm-charts/pull/129